CMakeLists.txt

```diff
@@ -67,7 +67,7 @@ if (USE_WASM_OPT)
         add_custom_command(
                 TARGET starling.wasm
                 POST_BUILD
-                COMMAND ${WASM_OPT} --strip-debug -O3 -o starling.wasm starling.wasm
+                COMMAND ${WASM_OPT} --strip-debug -O3 -g -o starling.wasm starling.wasm
```

Comment:
Title says `RelWithDebInfo` but this is the `Release` case I think -- should this be below instead?